package.json

```diff
@@ -40,7 +40,11 @@
     ]
   },
   "devDependencies": {
+    "msw": "^0.35.0",
     "ts-loader": "^9.2.6",
     "typescript": "^4.4.4"
+  },
+  "msw": {
+    "workerDirectory": "public"
   }
-}
+}
```

Comment:
Please, add empty line at the end - didn't spot it before.

Reply:
Done 👍